fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/digitalcash/DigitalCashSendFragment.java

```diff
@@ -155,13 +155,37 @@ private void setUpTheAdapter() throws KeyException {
           R.id.fragment_digital_cash_home,
           DigitalCashHomeFragment::newInstance);
     }
+
+    if (myArray == null) {
```

Comment:
As the method `digitalCashViewModel.getAttendeesFromTheRollCallList()` is annotated as @Nullable, so it wasn't very clear to me if it throwed the exception in case of null or not. Probably looking more into that method, it will throw a NoRollCallException, so I can change that with an assert clause